Procfile

```diff
@@ -1 +1 @@
-web: HUBOT_GITTER2_TOKEN=$HUBOT_GITTER2_TOKEN bin/hubot -a gitter2 --name ourbot
```

Comment:
This should be part of a separate commit or change - it is not part of this PR 😄